src/components/Release/Release.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useFetchSongs } from '../../services/hooks';
+import withList from '../HOCs/WithList/WithList';
+import Song from '../Songs/Song';
+import placeholder from '../../assets/placeholder.png';
+import { useParams } from 'react-router-dom';
+import style from './release.css';
+
+const Release = () => {
+  const { releaseTitle, releaseId, artist } = useParams();
+
+  const img = `http://coverartarchive.org/release/${releaseId}/front`;
+  const handleImgError = ({ target }) => (target.src = placeholder);
+  
+  const songsData = useFetchSongs(releaseId, artist);
+  const Songs = withList(Song);
+
+  return (
+    <section className={style.Release}>
+      <h2>{releaseTitle}</h2>
+      <img src={img} onError={handleImgError} alt={releaseTitle}/>
```

Comment:
I noticed that when using onError, the Network tab of the browser shows failed requests for the albums without images. So another way to handle this would be to pull the true/false value for coverart from the musicbrainz api, and then include some sort of logic to the effect of `coverArt ? use image : use placeholder image`